app/src/lib/editors/win32.ts

```diff
@@ -59,7 +59,7 @@ type WindowsExternalEditor = {
   readonly displayNamePrefix: string
 
   /** Value of the Publisher registry key that belongs to this editor. */
-  readonly publisher: string
+  readonly publisher: string | string[]
```

Comment:
Since this can be one or more, we should just use `string[]` and then 1-item arrays for those which only have 1 known publisher.
```suggestion
  readonly publishers: string[]
```